plugins/woocommerce/tests/php/includes/rest-api/Controllers/Version3/class-wc-rest-coupons-controller-tests.php

```
@@ -102,12 +104,12 @@ public function test_filter_coupons_by_status() {
 
 		$coupon_1 = \Automattic\WooCommerce\RestApi\UnitTests\Helpers\CouponHelper::create_coupon( 'dummycoupon-1', 'draft' );
 		$post_1   = get_post( $coupon_1->get_id() );
-		$coupon_2 = \Automattic\WooCommerce\RestApi\UnitTests\Helpers\CouponHelper::create_coupon( 'dummycoupon-2');
+		$coupon_2 = \Automattic\WooCommerce\RestApi\UnitTests\Helpers\CouponHelper::create_coupon( 'dummycoupon-2' );
```

PHPMD (2.15.0)

Avoid unused local variables such as '$coupon_2'. (Unused Code Rules)

(UnusedLocalVariable)